@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -307,19 +326,20 @@ export class TextElementsRenderer {
 
     /**
      * Places text elements for the current frame.
-     * @param tileTextElementsChanged Indicates whether there's been any change in the text elements
-     * to place since the last call to this method (last frame).
+     * @param dataSourceTileList List of tiles to be rendered for each data source.
+     * @param projection The view's projection.
+     * @param time Current frame time.
      * @param tilesChanged Indicates whether there's been any change in the tiles rendered since the
      * last call to this method (last frame).
-     * @param time Current frame time.
-     * @param frameNumber Current frame number.
      */
     placeText(
-        tileTextElementsChanged: boolean,
-        tilesChanged: boolean,
+        dataSourceTileList: DataSourceTileList[],
```

Comment:
Did you consider to add this to the ViewState? If yes, is there a reason it is here?

Reply:
I moved to ViewState all properties that are used throughout many functions in TextElementsRenderer. Since the m_viewState is a member of TextElementsRenderer, any function can use those properties without having to pass them as parameter.

For the tile list, only a couple of functions need to use it, so I think it's better to pass it explicitly only to the functions that need it.

Reply:
It seems also natural that when you call placeText, you pass as parameter the tiles containing the text.

Reply:
ok, fair enough